examples/dt/uni02beta/control-plane/service-values.yaml

```diff
@@ -43,7 +43,7 @@ data:
       customServiceConfig: |
         [DEFAULT]
         debug = true
-        enabled_share_protocols=cifs
+        enabled_share_protocols=nfs
```

Comment:
This line was already merged via https://github.com/openstack-k8s-operators/architecture/pull/282/files#diff-265437ff09c695319b6bd14a1ae97117b837cc248de62849ae4626e2542da355R58